src/i18n/en-US.json

```diff
@@ -15,5 +15,38 @@
     "remember": "Remember me",
     "forget": "Forget password?",
     "login": "Login"
+  },
+  "settingpage": {
+    "pageTitle": "Setting",
+    "oldPassword": "Enter old password",
+    "newPassword": "Enter new password",
+    "confirmPassword": "Enter new password again",
+    "changePassword": "Change Password",
+    "changelanguage": "中文",
```

Comment:
Don't set Chinese words in English setting file.

Reply:
done